sql/analyzer/check_constraints.go

```diff
@@ -29,6 +27,8 @@ import (
 
 // validateCreateCheck legal expressions for CREATE CHECK statements, including those embedded in CREATE TABLE
 // statements
+// TODO: validateCreateCheck doesn't currently do any type validation on the check and will allow you to create
+//       checks that will never evaluate correctly.
 func validateCreateCheck(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope) (sql.Node, error) {
```

Comment:
Probably should be named validateCheckConstraints, and get an updated comment about it applying to many DDL statements

Reply:
✅